public/js/sticky-sidebar.js

```diff
@@ -0,0 +1,38 @@
+/* global $ */
```

Comment:
I think that was the rewriting from `require` to `import`, if you revert that change we can make an issue for this work for the future.